vitest.config.shared.ts

```diff
@@ -0,0 +1,28 @@
+import * as vitest from 'vitest/config';
+
+export const base: vitest.ViteUserConfig = {
+  test: {
+    coverage: {
+      thresholds: { 100: true },
```

Comment:
Should we set thresholds for just `lines` and `branches`, since that's the convention we adopted previously?

Reply:
Done.